src/font-loader.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Polyfill loosely based on zenfonts
+ * https://github.com/zengabor/zenfonts/
+ * Copyright (c) 2015 Gabor Lenard
+ */
+
+/* global document */
+
+const testFonts = ['sans-serif', 'serif', 'monospace'];
+const testText = 'π+[.—_]imW12/?'; // FIXME: allow override
```

Comment:
We should change this to be a TODO which references an issue

Reply:
#3